ksql-parser/src/main/java/io/confluent/ksql/util/QueryMask.java

```diff
@@ -116,6 +151,32 @@ public String visitSingleStatement(final SingleStatementContext context) {
       return String.format("%s;", visit(context.statement()));
     }
 
+    @Override
+    public String visitInsertValues(final InsertValuesContext context) {
+      final StringBuilder stringBuilder = new StringBuilder("INSERT INTO ");
+
+      if (context.sourceName() != null) {
+        stringBuilder.append(" SOURCE");
```

Comment:
Why are we masking the source name? 

(Also, nit: there's a double-space between "INSERT INTO" and "SOURCE".)